templates/blog/post_detail.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ post.title }}</h1>
+  <hr>
+  <p>{{ post.content }}</p>
+  <hr>
+  {{ post.owner.first_name }} {{ post.owner.last_name }} {{ post.created_time }}
+  <hr>
+  {% if post.commentarys %}
```

Comment:
The check `{% if post.commentarys %}` will always evaluate to True because `post.commentarys` is a related manager. Use `{% if post.commentarys.exists %}` to check if there are any comments.